crates/bevy_winit/src/winit_windows.rs

```diff
@@ -31,16 +31,16 @@ impl WinitWindows {
                 winit::window::Fullscreen::Exclusive(match use_size {
                     true => get_fitting_videomode(
                         &event_loop.primary_monitor().unwrap(),
-                        window.width(),
-                        window.height(),
+                        window.physical_width(),
+                        window.physical_height(),
                     ),
                     false => get_best_videomode(&event_loop.primary_monitor().unwrap()),
                 }),
             )),
             _ => winit_window_builder
-                .with_inner_size(winit::dpi::LogicalSize::new(
-                    window.width(),
-                    window.height(),
+                .with_inner_size(winit::dpi::PhysicalSize::new(
+                    window.physical_width(),
```

Comment:
This causes a "bug" where we create a window at half of the desired size. This is because the `&Window` reference at this point hasn't had its scale factor set. We should use this instead:

```rust
.with_inner_size(winit::dpi::LogicalSize::new(
    window.logical_width(),
    window.logical_height(),
))
```

Reply:
done